interface/modules/custom_modules/oe-module-faxsms/src/Controller/EtherFaxActions.php

```diff
@@ -192,7 +192,11 @@ public function sendFax(): string
         $tag = $user['username'];
 
         if (empty($isContent)) {
-            $file = str_replace(["file://", "\\"], ['', "/"], realpath($file));
+            if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+                $file = str_replace(["file://", "\\"], ['', "/"], realpath($file));
+            } else {
+                $file = substr($file, 7);
+            }
             if (!$file) {
                 return xlt('Error: No content');
             }
```

Comment:
I can't merge this because it breaks on windows. Why?
realpath($file) is false because file path isn't valid i.e 'file://'. Previous versions of PHP must have put up with this but not now.

```
if (empty($isContent)) {
            if (str_starts_with($file, 'file://')) {
                // Remove the "file://" prefix
                $file = substr($file, 7);
            }
            $realPath = realpath($file);
            if ($realPath !== false) {
                $file = str_replace("\\", "/", $realPath);
            } else {
                return xlt('Error: No content');
            }
        }
```